packages/components/src/components/teaser/teaser.stories.ts

```diff
@@ -0,0 +1,352 @@
+import '../../solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+
+const { argTypes, parameters } = storybookDefaults('sd-teaser');
+const { overrideArgs } = storybookHelpers('sd-teaser');
+const { generateTemplate } = storybookTemplate('sd-teaser');
+
+export default {
+  title: 'Components/sd-teaser',
+  component: 'sd-teaser',
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'default',
+      value: `<slot-comp style="--slot-height: 48px; --slot-content: 'Main slot'"></slot-comp>`
+    },
+    {
+      type: 'slot',
+      name: 'media',
+      value: `<slot-comp slot='media' style="--slot-height: 48px; --slot-content: 'Media slot'"></slot-comp>`
+    },
+    {
+      type: 'slot',
+      name: 'headline',
+      value: '<h2 slot="headline">Remember: Always Use &lt;h&gt; Tags here!</h2>'
+    },
+    {
+      type: 'slot',
+      name: 'meta',
+      value: `<slot-comp slot='meta' style="--slot-height: 46px; --slot-content: 'Meta slot'"></slot-comp>`
+    }
+  ]),
+  argTypes,
+  parameters: { ...parameters }
+};
+
+/**
+ * This shows sd-teaser in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * Teaser in all possible variants.
+ */
+
+export const Variants = {
```

Comment:
Maybe you could extend it to `VariantAndInset` to see, that the thing with the border has always inset.